ansible/library/reduce_and_add_sonic_images.py

```diff
@@ -62,7 +62,7 @@ def install_new_sonic_image(module, new_image_url):
         exec_command(module, cmd="umount /tmp/tmpfs", ignore_error=True)
 
         exec_command(module,
-                     cmd="mount -t tmpfs -o size=1000M tmpfs /tmp/tmpfs",
+                     cmd="mount -t tmpfs -o size=1500M tmpfs /tmp/tmpfs",
```

Comment:
Let's change it to 1300 to match what we did in 201911 branch. The rest looks good.

Reply:
updated, thanks.